### ISSUE TITLE
OpenPandora forum broken

### ISSUE THREAD

**Issue by berserkov** [vranki/siilihai-client#8]
Hi!

I have tested the OpenPandora forum (https://pyra-handheld.com/boards/) with Siilihai 2.7.1-1 from the PPA. It is not working and I get the following error message

ClientLogic::ClientLogic(QObject*) Network connected: true
virtual void ClientLogic::launchSiilihai() Loaded settings from  "/home/vladimir/.local/share/Siilihai/Siilihai/siilihai_settings.ini"
virtual void ClientLogic::launchSiilihai() Using data files under  "/home/vladimir/.local/share/Siilihai/Siilihai"
void UpdateEngine::setState(UpdateEngine::UpdateEngineState) "OpenPandora Boards" "Missing Parser"  ->  "Idle"
[6641:6662:0330/093524.104005:ERROR:nss_util.cc(808)] After loading Root Certs, loaded==false: NSS error code: -8018
virtual void ClientLogic::setState(ClientLogic::SiilihaiState) Offline changed:  false
void ParserManager::offlineChanged(bool) false
virtual void ClientLogic::setState(ClientLogic::SiilihaiState) Login
virtual void ClientLogic::setState(ClientLogic::SiilihaiState) Startsync
virtual void ClientLogic::syncFinished(bool, QString) true
void ClientLogic::updateForum(ForumSubscription*) updating now  "Subscription to 88 (OpenPandora Boards) su 0 bu 0 ss 0 bs 0" ()
virtual void UpdateEngine::updateForum(bool) Updating "OpenPandora Boards"
void UpdateEngine::setState(UpdateEngine::UpdateEngineState) "OpenPandora Boards" "Idle"  ->  "Updating"
virtual void ClientLogic::setState(ClientLogic::SiilihaiState) Ready
QString TapaTalkEngine::getValueFromStruct(QDomElement, QString) Error: node name is not 'value'
void UpdateEngine::setState(UpdateEngine::UpdateEngineState) "OpenPandora Boards" "Updating"  ->  "Error"
void ClientLogic::updateForum(ForumSubscription*) updating now  "Subscription to 88 (OpenPandora Boards) su 0 bu 0 ss 0 bs 0" ()
virtual void UpdateEngine::updateForum(bool) Updating "OpenPandora Boards"
void UpdateEngine::setState(UpdateEngine::UpdateEngineState) "OpenPandora Boards" "Error"  ->  "Updating"
QString TapaTalkEngine::getValueFromStruct(QDomElement, QString) Error: node name is not 'value'
void UpdateEngine::setState(UpdateEngine::UpdateEngineState) "OpenPandora Boards" "Updating"  ->  "Error"

I assume that the forum configuration might be broken. I cannot add the forum manually because Siilihai is shortening the URL of the forum from https://pyra-handheld.com/boards/forums/ to https://pyra-handheld.com/

BR

berserkov

**Comment by vranki** [comment]
Hi, i checked this.
First, the board has changed from http to https, so I changed the forum URL. You need to unsubscribe and subscribe to get the new URL in use.

Another issue is that I'm getting permission denied error from the tapatalk endpoint:

https://boards.openpandora.org/mobiquo/mobiquo.php


```

<?xml version="1.0" encoding="UTF-8"?>
<methodResponse>
<params>
<param>
<value><struct>
<member><name>result</name>
<value><boolean>0</boolean></value>
</member>
<member><name>result_text</name>
<value><base64>WW91IGFyZSBub3QgbG9nZ2VkIGluIG9yIHlvdSBkbyBub3QgaGF2ZSBwZXJtaXNzaW9uIHRvIGRvIHRoaXMgYWN0aW9uLg==</base64></value>
</member>
<member><name>error</name>
<value><base64></base64></value>
</member>
</struct></value>
</param>
</params>
</methodResponse>
```

The encoded text is: "You are not logged in or you do not have permission to do this action."

I think this is a configuration error on the forum. Could you ask the forum admins to check this?

**Comment by berserkov** [comment]
Hi!

Thanks a lot for the quick reply! I can confirm, that after a re-subscription the authentication fails.
Do you approximately know, which is the necessary configuration change?

**Comment by vranki** [comment]
Nope, I haven't hosted any TapaTalk forums so I don't know about it's configuration. Maybe it's some kind of permission setting denying non-authenticated users to access the forum via TapaTalk.

**Comment by EvilDragon** [comment]
Hi,

EvilDragon here (the admin).

The only thing in the config I can see that could be the issue was:
Two Factor Authentication for Tapatalk users: Enforced

I've now switched it to bypass. Otherwise, there's not really much that could affect it in my opinion.

Please retest.

And let me know how I can assist in debugging!

**Comment by berserkov** [comment]
Unfortunately it seems to be still not working :(
The error message is still the same but I think that there is still some settings mismatch.

**Comment by EvilDragon** [comment]
Well, attached are all the settings that the Tapatalk plugin offers.
Not really much that could interfere, I think.

![settings](https://user-images.githubusercontent.com/815110/39674284-c6328d0e-5149-11e8-81a5-50a84e3b3362.png)

Could it be that the plugin needs to be of a more recent version?
It works fine with Tapatalk though.
